### ISSUE TITLE
Support shortcuts in place of apps for an action

### ISSUE THREAD

**Issue by goodevilgenius** [finnmglas/Launcher#45]
**Is your feature request related to a problem? Please describe.**

Some actions I want to perform from my homescreen aren't simply opening apps, but are available as shortcuts

**Describe the solution you'd like**

It would be useful to bind events to shortcuts, instead of just apps.

**Describe alternatives you've considered**

Haven't been able to come up with any.

**Additional info**

E.g., Google app has a "Sound Search" shortcut. Creating one of these shortcuts on a supported launcher adds an icon that, when clicked, launches the Google app and starts listening for music, which it identifies.

Tasker allows you to add shortcuts to a task. When the task launches the shortcut, it does the same thing as clicking on the icon from a stock launcher.

Launcher could support shortcuts in the same way that Tasker does. It need not add a visible icon, but when the chosen action is performed, instead of launching and app, it sends the Intent for the shortcut.

**Comment by esmoak** [comment]
It would be nice if you could swipe to "view all apps" at least.

**Comment by finnmglas** [comment]
I totally agree! That's why I recently implemented this feature (in the last week) ^^

I will make sure to publish the release (v1.3.0) within the next 10 days. It contains the feature you are looking for.

I also implemented a search feature in the list, as scrolling can become tedious after a while.
